Extensions/one_click_postage.js

```diff
@@ -974,7 +833,6 @@ XKit.extensions.one_click_postage = new Object({
 						XKit.extensions.one_click_postage.user_on_box = true;
 						XKit.extensions.one_click_postage.open_menu($(this), false, true);
 						$('#x1cpostage_tags').focus();
-						break;
```

Comment:
I like keeping the `break` here so that it's less disruptive to add more cases in the future